custom_components/zwave_mqtt/climate.py

```diff
@@ -40,23 +41,47 @@
 
 ATTR_FAN_ACTION = "fan_action"
 
+
+class ThermostatMode(IntEnum):
```

Comment:
We should move this to the library.

Reply:
Yeah we need a close look at all files what can be turned into more enums/constants in the lib.